### DISCUSSION TITLE
Problems with low latency video

### DISCUSSION THREAD

**Discussion by jorge-msp** [roleoroleo/yi-hack_ha_integration#159] [Q&A]
Hello everyone, I've just hacked a Yi camera 2k Pro with the yi-hack-Allwinner-v2 and successfully added the camera to my home HAOS. The camera streaming works, but with low latency and sometimes with distorted video.

The signal strength measured on the IP-CAM is always around 85%-95% so I guess the problem comes from elsewhere.

I would appreciate any suggestion to help me out.
Thanks in advance

**Comment by roleoroleo** [comment]
If you are using the last released version, there are some pending fixes I have to release.
Maybe the problem is already fixed.
